contents/blog/multi-product-pricing.md

```diff
@@ -0,0 +1,73 @@
+---
+date: 2023-04-04
+title: What we've learned so far about multi product pricing, and our future plans
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+category: CEO diaries
+---
+
+PostHog started as a single product company. We just sold product analytics with a bunch of large extra features thrown in - session replay, feature flagging, experimentation, and even a Customer Data Platform.
+
+We charged by event ingested. Obviously, we wound up with abuse - a handful of high volume users would send zero events, but would use our Javascript snippet to do just session replay, to store _millions_ of recordings.
+
+We decided a multi product pricing strategy had long term advantages, and - to be frank - we had to stop people abusing recording, and we didn't want to limit it for users already paying us a healthy amount.
+
+## It's a way to take extra products seriously
+
+The most important thing, but the one we didn't anticipate, was charging for session recording very quickly led to more usage.
+
+Eh? 
+
+Positioning. If you see two products listed side-by-side on a website, you know each is being treated as a first class citizen. Beforehand, we were "product analytics++". That meant no one was taking our session replay feature seriously.
+
+The moment we started charging, more users realized the feature existed, the more seriously they took it, and the more demanding their feedback. Paying users are less bug tolerant!
+
+We put a small team on it, who quickly iterated with customers and improved the quality.
+
+Growth of session replay took off - it now has similar daily usage to product analytics.
+
+## Focusing on paying customers can improve the free product
+
+Intuitively, if someone is paying, they will have a higher expectation.
+
+However, the secondary effect we didn't anticipate is that fixing the smaller issues can lead to much more usage from both paid and free users. It turns out that in 2021, a lot of UX weirdness and buginess we had was affecting all users of our platform, but our free users weren't complaining that loudly.
+
+As context, PostHog spent the first year and a half prioritizing our open source project - we wanted to build an inbound sales model, so we could double down on product rather than investing in cold calling. Leverage, baby. This meant we'd need a great free product and a reasonably large community. After 18 months, we were ready to focus on our first paid product - product analytics.
```

Comment:
```suggestion
We spent the first year and a half of PostHog prioritizing our open-source project – we wanted to build an inbound sales model, so we could double down on product rather than investing in cold calling. This meant we'd need a great free product and a reasonably large community. After 18 months, we were ready to focus on our first paid product – product analytics.
```

Reply:
Again, James here was messier but more authentic? ('Leverage baby')